noxfiles/dev_nox.py

```diff
@@ -44,3 +44,29 @@ def quickstart(session: nox.Session) -> None:
     build(session, "dev")
     session.notify("teardown")
     run_infrastructure(datastores=["mongodb", "postgres"], run_quickstart=True)
+
+
+@nox.session()
+@nox.parametrize(
+    "script_name",
+    [
+        nox.param("example_script", id="example_script"),
+    ],
+)
+def run_script(session: nox.Session, script_name: str) -> None:
+    """
+    Run a script from the scripts/ directory. This command will not run
+    the server automatically.
+    """
+    posargs = session._runner.posargs
```

Comment:
why is this `session._runner.posargs` instead of `session.posargs`? I just want to make sure I'm not missing anything here, but `session.posargs` is the standard way to access the posargs

Reply:
Will update — this was the result of me digging around in the interpreter and seeing what's available. The `nox` docs seem quite light on detail here

Reply:
Thanks for rectifying this